automations/python/models/label_group.py

```diff
@@ -14,6 +19,8 @@ def __init__(
         is_required=False,
         **kwargs,
     ):
+        """Initialise LabelGroup."""
```

Comment:
Should be `#` because it's not documenting anything but rather demarcating the upcoming block.
```suggestion
        # Initialise LabelGroup.
```

Reply:
We can actually remove it entirely, I think it is left over from when I was still trying to decide if any of the D1xx level rules should be kept.